metric_learn/itml.py

```diff
@@ -181,16 +181,16 @@ class ITML_Supervised(_BaseITML, TransformerMixin):
 
   Attributes
   ----------
-  bounds_ : `list` of two numbers
+  bounds_ : array-like, shape=(2,)
       Bounds on similarity, aside slack variables, s.t. ``d(a, b) < pos`` for
       all given pairs of similar points ``a`` and ``b``, and
       ``d(c, d) > neg`` for all given pairs of dissimilar points ``c`` and
       ``d``, with ``bounds=[pos, neg]``, and ``d`` the learned distance. If
```

Comment:
I keep saying ``bounds=[pos, neg]`` here because it can be still be a list here, and I think this is more general than ``bounds=array([pos, neg])`` which would make think more of a numpy array only